src/server/dfly_main.cc

```diff
@@ -594,7 +595,20 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void print_basic_usage_info() {
+  std::cout << "* Default logdir is: \"/tmp/\". It can be overwritten with flag --log_dir=[path]\n";
+  std::cout << "* Logfile name format is: ";
+  std::cout << "dragonfly.[machine_name].[user_name].[WARNING|INFO|ERROR].[date and time]\n";
```

Comment:
I think explaining the format it too much. I have never seen a python program explaining its logger config.

Reply:
+1, that's kind of my point :)